gae_discourse_client/content.py

```diff
@@ -0,0 +1,48 @@
+"""Module for accessing and creating content through the Discourse API"""
+
+from google.appengine.ext import ndb
+
+
+class Error(Exception):
+    pass
+
+
+class ContentClient(object):
+    """An API client for interacting with Discourse for content-related actions"""
+
+    def __init__(self, api_client, category_client):
+        self._api_client = api_client
+        self._category_client = category_client
+
+    @ndb.tasklet
+    def get(self, category_id=None, page=0):
```

Comment:
Fixed the name -- we do have a separate client for the category, but I think treating content and categories separately is good (although I don't think we need to make a distinction between topics and posts).